startlotto.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>로또 번호 생성기</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <span class="headingtext">로또 번호 생성기</span>
+            <span class="headingtextsub">행운의 번호를 생성해보세요!</span>
+        </header>
+        <div class="bodybox">
+            <div class="makelottobox">
+                <span class="makenumtext">번호 생성</span>
+                <div class="circlebox">
+                    <span class="circle1">?</span>
+                    <span class="circle2">?</span>
+                    <span class="circle3">?</span>
+                    <span class="circle4">?</span>
+                    <span class="circle5">?</span>
+                    <span class="circle6">?</span>
```

Comment:
`class`와 `id`를 적절히 사용해서 중복되는 디자인은 하나의 css로 묶을 수 있어요.
```suggestion
                    <span class="circle" id="circle1">?</span>
                    <span class="circle" id="circle2">?</span>
                    <span class="circle" id="circle3">?</span>
                    <span class="circle" id="circle4">?</span>
                    <span class="circle" id="circle5">?</span>
                    <span class="circle" id="circle6">?</span>
```

이런식으로 지정하고 class 선택자에는 기본 디자인을 적용하고 circle1 ~ 6까지는 배경색만 입혀줄 수 있어요!

Reply:
class와 id를 같이 쓸 수 있는 방법을 생각도 못했었습니다. 조언 감사합니다!!